js/src/admin/components/StatusWidget.js

```diff
@@ -37,6 +37,17 @@ export default class StatusWidget extends DashboardWidget {
     return items;
   }
 
+  toolsItems() {
+    const items = new ItemList();
```

Comment:
```suggestion
    const items = new ItemList<Mithril.Children>();
```

(Also need to add `import type Mithril from 'mithril';` to the top of the file)

Reply:
But this is a JS file, not a TS file

Reply:
Ermmmmm you saw nothing! It's been a long night, sorry. 🙈